apps/crn-frontend/src/analytics/engagement/export.ts

```diff
@@ -1,14 +1,34 @@
-import { EngagementDataObject } from '@asap-hub/model';
+import { EngagementDataObject, EngagementPerformance } from '@asap-hub/model';
+import { utils } from '@asap-hub/react-components';
 
-export const engagementToCSV = (data: EngagementDataObject) => ({
-  Team: data.name,
-  'Team Status': data.inactiveSince ? 'Inactive' : 'Active',
-  'Inactive Since': data.inactiveSince ? data.inactiveSince.split('T')[0] : '',
-  Members: data.memberCount,
-  Events: data.eventCount,
-  'Total Speakers': data.totalSpeakerCount,
-  'Unique Speakers (All Roles): Value': data.uniqueAllRolesCount,
-  'Unique Speakers (All Roles): Percentage': `${data.uniqueAllRolesCountPercentage}%`,
-  'Unique Speakers (Key Personnel): Value': data.uniqueKeyPersonnelCount,
-  'Unique Speakers (Key Personnel): Percentage': `${data.uniqueKeyPersonnelCountPercentage}%`,
-});
+export const engagementToCSV =
+  (performance: EngagementPerformance) => (data: EngagementDataObject) => ({
+    Team: data.name,
+    'Team Status': data.inactiveSince ? 'Inactive' : 'Active',
+    'Inactive Since': data.inactiveSince
+      ? data.inactiveSince.split('T')[0]
+      : '',
+    Members: data.memberCount,
+    'Presentations: Value': data.eventCount,
+    'Presentations: Average': utils.getPerformanceText(
+      data.eventCount,
```

Comment:
```suggestion
    'Events: Value': data.eventCount,
    'Events: Average': utils.getPerformanceText(
      data.eventCount,
```

I believe we could still call this 'Events' since we're calling it this way on the table. I believe it started as 'Presentation' and then it became 'Events' but I think maybe it wasn't updated in all places ?

<img width="1720" alt="Screenshot 2024-09-06 at 07 54 37" src="https://github.com/user-attachments/assets/a4f3021b-997c-4d09-ae7c-e2d730dfa38d">